automation/identifiers/MainTable.ts

```diff
@@ -0,0 +1,4 @@
+export default class MainTable {
```

Comment:
please make naming more clear, 'main' is not direct to any thoughts about what a table is it (especially that you stores few different tables)